connect-examples/server-library-example/client/src/main/scala/org/example/CustomTableBuilder.scala

```diff
@@ -0,0 +1,134 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.example
```

Comment:
It's a little weird to have `org.example` in ASF repository.

Technically, we always use `org.apache.` prefix.
```
$ git grep '^package org' | grep -v org.apache | awk '{print $NF}' | sort | uniq -c
```

Reply:
Thanks, I've updated `org.example` to a more descriptive `org.apache.connect.examples.serverlibrary`